scripts/generate-AUTHORS.py

```diff
@@ -232,14 +136,6 @@ def main():
 		if line == "zapman <unknown>":
```

Comment:
We have no email for the guy. Git shows <unknown>, I like to show nothing at all in that case. That's pretty much it. I could make it more general, I suppose...